packages/config/config/devices/0x0460/qnsw-001P16.json

```
@@ -73,102 +99,220 @@
 		{
 			"#": "23",
 			"label": "O1 No/Nc",
-			"description": "0",
+			"description": "The set value determines the relay contact type for output O (O1). The relay contact type can be normally open (NO) or normally closed (NC).",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+            "allowManualEntry": false,
+			"options": [
+				{
+					"label": "NO",
+					"value": 0
+				},
+				{
+					"label": "NC",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "25",
 			"label": "Set Timer Units to Seconds Or Milliseconds For O1",
```

```suggestion
			"label": "O1: Auto-On/Off Timer Unit",
```
makes the description unnecessary